src/main/java/com/google/sps/servlets/MenteeSignupServlet.java

```diff
@@ -32,6 +33,45 @@
 @WebServlet("/mentee-signup")
 public class MenteeSignupServlet extends HttpServlet {
 
+  /**
+   * Gets majors from database and returns as JSON.
+   */
```

Comment:
Are you going to need this GetMajors function for both Mentors and Mentees? If so you should pull this into a util function so that you can share the logic for both types of user, rather than copy/pasting repeated code.

Reply:
Ah I see now that you are using this for both the mentor and mentee sign up. I'm not sure if it makes sense for this logic to be in the MenteeSignupServlet. Would it be possible to have a more generic SignupServlet that can perform this shared behavior?

Reply:
Cool, did that! Tell me if it's good now.